set.mm

```diff
@@ -837767,6 +838014,1624 @@ have GLB (expanded version).  (Contributed by Zhi Wang,
+        SRUGUHWCWFCFUOOOWBWEBCUOOONBUPRNPZUEPRZUGPRWGUHPRWGMPZRZWHUDPORWJUCPZRW
+        HWKRUQWHUFPRFURROOUSUTVAVAVAVAVAZUTZUQTTULANUBUABCDEMUCUEFGVSHUFUHUGUDI
+        JKLAVSVBVCVTWMUKVDVSTVEVTTVEUIMTTUIPZSRWISRZVFNUJWIQRZVGVGZWQWGWORUJPZW
+        ORWNQROWGWRWPOVFUTUQUKNUJMUIVHVIVQVRCFUMVJBCUMVJVKZUKVSTVLVMUAUBVSVSWLW
+        SWSVNVOVP $.
+    $}
+
+    fuco1.o $e |- ( ph -> <. O , P >. = ( <. C , D >. o.F E ) ) $.
+    fuco1.w $e |- ( ph -> W = ( ( D Func E ) X. ( C Func D ) ) ) $.
+    $( The object part of the functor composition bifunctor.  (Contributed by
+       Zhi Wang, 29-Sep-2025.) $)
+    fuco1 $p |- ( ph -> O = ( o.func |` W ) ) $=
+      ( cv cfv c1st co cvv vu vv vf vk vl vm vr vb va vx cop cres c2nd cnat cbs
+      ccofu cco cmpt cmpo csb wceq fucofval wi cxp fucoelvv opelxp1 syl opelxp2
+      wcel opth1g syl2anc mpd ) AHDUKZUPJULZUAUBJJUCUAPZUMQZRQUDVORQZRQUEVQUMQU
+      FUBPZUMQZRQUGVRRQZRQUHUIVQVTCGUNSSVPVSBCUNSSUJBUOQUJPZUFPQZUHPQWAUIPQWAUC
+      PQZWBUEPSQWCUDPZQWBWDQUKWBUGPQGUQQSSURUSUTUTUTUTUTUSZUKVAZHVNVAZAUJUBUABC
+      EFUCUDUFGIJVMUGUIUHUEKLMNOVBAHTVIZDTVIZWFWGVCAVMTTVDVIZWHABCEFGIVMKLMNVEZ
+      HDTTVFVGAWJWIWKHDTTVHVGHDVNWETTVJVKVL $.
+
+    $( The object part of the functor composition bifunctor maps
+       ` ( ( D Func E ) X. ( C Func D ) ) ` into ` ( C Func E ) ` .
+       (Contributed by Zhi Wang, 29-Sep-2025.) $)
+    fucof1 $p |- ( ph -> O : W --> ( C Func E ) ) $=
+      ( cfunc co wf ccofu cres cxp rescofuf fuco1 reseq2d eqtrd feq12d mpbiri )
+      AJBGPQZHRCGPQBCPQUAZUHSUITZRBCGUBAJUIUHHUJAHSJTUJABCDEFGHIJKLMNOUCAJUISOU
+      DUEOUFUG $.
+
+    $( The morphism part of the functor composition bifunctor.  (Contributed by
+       Zhi Wang, 29-Sep-2025.) $)
+    fuco2 $p |- ( ph -> P = ( u e. W , v e. W |->
+ [_ ( 1st ` ( 2nd ` u ) ) / f ]_
+ [_ ( 1st ` ( 1st ` u ) ) / k ]_ [_ ( 2nd ` ( 1st ` u ) ) / l ]_
+ [_ ( 1st ` ( 2nd ` v ) ) / m ]_ [_ ( 1st ` ( 1st ` v ) ) / r ]_
+ ( b e. ( ( 1st ` u ) ( D Nat E ) ( 1st ` v ) ) ,
+   a e. ( ( 2nd ` u ) ( C Nat D ) ( 2nd ` v ) ) |->
+ ( x e. ( Base ` C ) |-> ( ( b ` ( m ` x ) )
+ ( <. ( k ` ( f ` x ) ) , ( k ` ( m ` x ) ) >. ( comp ` E ) ( r ` ( m ` x ) ) )
+ ( ( ( f ` x ) l ( m ` x ) ) ` ( a ` x ) ) ) ) ) ) ) $=
+      ( ccofu cres wceq cv c2nd cfv c1st cnat co cbs cop cco cmpt cmpo fucofval
+      csb wa cvv wb cxp fucoelvv opelxp1 syl opelxp2 opthg syl2anc mpbid simprd
+      wcel ) ANUFPUGZUHZGDCPPJDUIZUJUKZULUKKVQULUKZULUKTVSUJUKLCUIZUJUKZULUKQVT
+      ULUKZULUKSRVSWBFMUMUNUNVRWAEFUMUNUNBEUOUKBUIZLUIUKZSUIUKWCRUIUKWCJUIUKZWD
+      TUIUNUKWEKUIZUKWDWFUKUPWDQUIUKMUQUKUNUNURUSVAVAVAVAVAUSZUHZANGUPZVOWGUPUH
+      ZVPWHVBZABCDEFHIJKLMOPWIQRSTUAUBUCUDUEUTANVCVNZGVCVNZWJWKVDAWIVCVCVEVNZWL
+      AEFHIMOWIUAUBUCUDVFZNGVCVCVGVHAWNWMWONGVCVCVIVHNGVOWGVCVCVJVKVLVM $.
+
+    $d P u v $.
+    $( The morphism part is a function on the Cartesian product.  (Contributed
+       by Zhi Wang, 29-Sep-2025.) $)
```

Comment:
Maybe it's better to repeat : _"The morphism part **of the functor composition bifunctor**"_, to give context.